From cf68108ad4958fc04af3776eed7199b880ca938c Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Sun, 7 Oct 2018 18:23:28 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb: fix up the polarity for
 syr8xx vsel gpio

Set vsel gpio low to disable vdd_npu.
Set vsel gpio high to enable vdd_npu.
The vsel pin is pull up by hw default.

Change-Id: I86ef1f2d6d2a0dd26a128cba1e52a140e6844291
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index ff7f101ac519..6c1bb4e67835 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -243,7 +243,7 @@
 		regulator-min-microvolt = <750000>;
 		regulator-max-microvolt = <950000>;
 		regulator-ramp-delay = <1000>;
-		fcs,suspend-voltage-selector = <1>;
+		fcs,suspend-voltage-selector = <0>;
 		regulator-boot-on;
 		regulator-state-mem {
 			regulator-off-in-suspend;
-- 
2.35.3

